Marsiple/App/New Group/PhotoCollectionViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PhotoCollectionViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class PhotoCollectionViewController: UICollectionViewController {
+    private let photos: [UIImage]
+    
+    init(photos: [UIImage]) {
+        self.photos = photos
+        super.init(collectionViewLayout: UICollectionViewFlowLayout.init())
```

Comment:
We're not really using this `convention` anywhere else in the app, so lets ditch it here and use the `"shortened"` initialiser - `UICollectionViewFlowLayout.init()` -> `UICollectionViewFlowLayout()`